### ISSUE TITLE
CLI 1.7 ng update command is broken (was "CLI 1.7 fails to update me to 6")

### ISSUE THREAD

**Issue by StephenFluin** [angular/angular-cli#10254]
### Versions
start with
```
Angular CLI: 1.7.4
Node: 9.2.0
OS: linux x64
Angular:

### Repro steps
1. `ng new update-test3'
2. `ng update --next`


### Observed behavior
```
Error: Cannot read property '1' of null
Cannot read property '1' of null
```

### Desired behavior
It should update my package.json to the v6 CLI or tell me what to do next to get to v6.

**Comment by hansl** [comment]
```
"CLI 1.7 does not support an automatic v6 update. Manually install @angular/cli via your package manager, then run the update migration schematic to finish the process.

npm install @angular/cli@^6.0.0
ng update @angular/cli --migrate-only --from=1"
```

**Comment by hansl** [comment]
Fixed by PR #10310

**Comment by mahdikianirad** [comment]
It happened for me because I had package-lock.json still in the app.
remove the package-lock.json then
ng update @angular/cli
